### ISSUE TITLE
cant go online on valorant again

### ISSUE THREAD

**Issue by tiaohappy** [ParvinEyvazov/valoline#1]
the last time i used valoline i set on "offline" mode and now after a long time i cant open it for some reason and all my riots games are offline and unavaliable service so i cant play with my friends. if there is a away to open it to put it on online mode let me know

**Comment by ParvinEyvazov** [comment]
Hi @tiaohappy, please run your Command Prompt (or cmd) as an administrator, and paste the command below to fix it


`netsh advfirewall firewall add rule name="lolchat" dir=out remoteport=5223 protocol=TCP action=block`

**Comment by tiaohappy** [comment]
@ParvinEyvazov i tried it but it didnt work it is still saying "unavaliable service" at the friend list *i ve tried while the game was closed and running

**Comment by tiaohappy** [comment]
is there a way to open the valoline program to switch to online mode? every time that i try to open it it closes. when i double click it to open he appears for a brief time and it closes

**Comment by tiaohappy** [comment]
i have tried this command to
**netsh advfirewall firewall delete rule name="lolchat"**

**Comment by ParvinEyvazov** [comment]
@tiaohappy  Also the last command didn't worked?

**Comment by ParvinEyvazov** [comment]
Because Valoline is basically running `netsh advfirewall firewall delete rule name="lolchat"` command behind to go online

**Comment by ParvinEyvazov** [comment]
I also could not open the application. I will try to create a new version by fixing this problem.

**Comment by tiaohappy** [comment]
@ParvinEyvazov yes this command       netsh advfirewall firewall delete rule name="lolchat"   also didnt work i have tried both but nothing

**Comment by tiaohappy** [comment]
> I also could not open the application. I will try to create a new version by fixing this problem.

ohh I really appreciate it if you can make a new version!! ps: sorry for taking so long to reply late at night its because i live in japan 😁

**Comment by ParvinEyvazov** [comment]
@tiaohappy Oh for sure, no problem 😊

Also please make sure you start your Command Prompt as an Administrator. Also thank you for your report.

**Comment by tiaohappy** [comment]
The thing that i dont understand: if valoline is basically a shortcut-program that executes those two commands: *netsh advfirewall firewall add rule name="lolchat" dir=out remoteport=5223 protocol=TCP action=block* and *netsh advfirewall firewall delete rule name="lolchat"* why when i use them in CMD (as an Administrator) it doesn't work!!!! Every day im trying these two commands but nothing changes.... i hope with the new version it fixes, because I always used valoline and it worked perfectly.

**Comment by ParvinEyvazov** [comment]
@tiaohappy  Are you using windows 10 or 11?

**Comment by tiaohappy** [comment]
im using windows 11

**Comment by tiaohappy** [comment]
Oh could it be that it didn't work anymore because I switched to windows 11? because before it always worked

**Comment by sprakll** [comment]
im also having the same problem, i typed in the command but im still offline

**Comment by sprakll** [comment]
when will this be fixed?

**Comment by pherbxe32** [comment]
bro i didnt know you guys had this problem too i thought i was the only one on the planet lol i already had this problem for over a year so i stopped playing valorant now that i decided to play again and i still have the same problem . I even tried contacting riot support but nothing they said worked for me...

**Comment by pherbxe32** [comment]
I saw several videos about it but I didn't find anything until I found this one from valoline and I remembered that I had used it but I wasn't sure, so I came here and found you guys talking about it too! It's a relief for me!! hope this will be fixed..

**Comment by ParvinEyvazov** [comment]
Hey guys, I am investigating this problem, I think it is somehow related to new updates of Valorant. I am trying to find a way to solve it, thank you for your patience.

**Comment by pherbxe32** [comment]
> Hey guys, I am investigating this problem, I think it is somehow related to new updates of Valorant. I am trying to find a way to solve it, thank you for your patience.

Thanks man !! it would be so helpful
